src/KeyConnector/KeyConnector.csproj

```diff
@@ -18,19 +20,19 @@
     <PackageReference Include="Google.Cloud.Kms.V1" Version="3.7.0"/>
     <PackageReference Include="IdentityServer4.AccessTokenValidation" Version="3.0.1"/>
     <PackageReference Include="JsonFlatFileDataStore" Version="2.4.2"/>
-    <PackageReference Include="Microsoft.EntityFrameworkCore.Design" Version="7.0.14">
+    <PackageReference Include="Microsoft.EntityFrameworkCore.Design" Version="8.0.2">
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
-    <PackageReference Include="Microsoft.EntityFrameworkCore.Relational" Version="7.0.14"/>
-    <PackageReference Include="Microsoft.EntityFrameworkCore.Sqlite" Version="7.0.14"/>
-    <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="7.0.14"/>
+    <PackageReference Include="Microsoft.EntityFrameworkCore.Relational" Version="8.0.2"/>
+    <PackageReference Include="Microsoft.EntityFrameworkCore.Sqlite" Version="8.0.2"/>
+    <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="8.0.2"/>
     <PackageReference Include="MongoDB.Driver" Version="2.22.0"/>
-    <PackageReference Include="Npgsql.EntityFrameworkCore.PostgreSQL" Version="7.0.11"/>
+    <PackageReference Include="Npgsql.EntityFrameworkCore.PostgreSQL" Version="8.0.0"/>
     <PackageReference Include="Pkcs11Interop" Version="5.1.2"/>
-    <PackageReference Include="Pomelo.EntityFrameworkCore.MySql" Version="7.0.0"/>
-    <PackageReference Include="Serilog.AspNetCore" Version="7.0.0"/>
-    <PackageReference Include="Serilog.Settings.Configuration" Version="7.0.1"/>
+    <PackageReference Include="Pomelo.EntityFrameworkCore.MySql" Version="8.0.0"/>
```

Comment:
ℹ️ I jammed in as many major updates as I could, but this one and PGSQL need special attention.